cmd/podman/containers/commit.go

```diff
@@ -105,9 +104,6 @@ func commit(cmd *cobra.Command, args []string) error {
 	if !commitOptions.Quiet {
 		commitOptions.Writer = os.Stderr
 	}
-	if len(commitOptions.Changes) > 0 {
-		commitOptions.Changes = handlers.DecodeChanges(commitOptions.Changes)
```

Comment:
Why is this being removed? It looks like it ought to do something

Reply:
This thing is called several times in the same code path so I dropped one instance